### PR TITLE
Create a mailmap file for committers

### PR DESCRIPTION
**Pull request by DelazJ**
Highly inspired by https://github.com/qgis/QGIS/pull/55547. Thanks @gacarrillor

### CODE REVIEW

**Comment by tomchadwin** [comment]
As in https://github.com/qgis/QGIS/pull/55547, tom.chadwin@nnpa.org.uk is no longer live, and I'm now tomchadwin@astuntechnology.com. Thanks, @DelazJ!

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+
+Abdul Raheem Siddiqui <53625184+ar-siddiqui@users.noreply.github.com>  Abdul Raheem Siddiqui <53625184+ar-siddiqui@users.noreply.github.com>
+
+Adrien Glauser <adrien.glauser@gmail.com>  Adrien <adrien.glauser@gmail.com>
+Adrien Glauser <adrien.glauser@gmail.com>  Adrien <mrnycticorax@gmail.com>
+
+Akbar Gumbira <akbargumbira@gmail.com>  Akbar Gumbira <akbargumbira@gmail.com>
+
+Alessandro Pasotti <elpaso@itopen.it>  Alessandro Pasotti <apasotti@gmail.com>
+Alessandro Pasotti <elpaso@itopen.it>  Alessandro Pasotti <elpaso@itopen.it>
```
Comment (gacarrillor) [inline .mailmap:13]: You could get rid of this kind of lines where you're mapping exactly the same name/e-mail pair to the left and to the right. They have no useful meaning in this mapping.

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+Erik Meerburg <erik@geo-academie.nl>  Erik Meerburg <erik@geo-academie.nl>
+
+Étienne Trimaille <gustrimaille@yahoo.fr>  Etienne Trimaille <gustrimaille@ahoo.fr>
+Étienne Trimaille <gustrimaille@yahoo.fr>  Etienne Trimaille <gustrimaille@yahoo.fr>
+Étienne Trimaille <gustrimaille@yahoo.fr>  Étienne Trimaille <gustrimaille@yahoo.fr>
+
+Felix Feckler <felix.feckler@gmx.de>  felixfeckler <55293275+felixfeckler@users.noreply.github.com>
+Felix Feckler <felix.feckler@gmx.de>  felixfeckler <felix.feckler@gmx.de>
+
+Fran <franka1986@gmail.com>  Fran <franka1986@gmail.com>
```
Comment (All4Gis) [inline .mailmap:151]: Can you change the name to use exactly the same in this two lines?like this
```
Fran Raga <franka1986@gmail.com> Fran Raga <franka1986@gmail.com>
Fran Raga <All4Gis@users.noreply.github.com> Fran Raga <All4Gis@users.noreply.github.com>
```
Thanks a lot

File: .mailmap (hunk 1 above)
Comment (DelazJ) [reply .mailmap:13]: @gacarrillor Only for people that have more than one entry or also those that are displayed only once? e.g. line 10

Comment (gacarrillor) [reply .mailmap:13]: For both of them.
What this states is that name/e-mail on the right-hand side should be mapped to name/e-mail on the left-hand side, which is already the case.

**Pull request merged**